tests/parsing/test_full_telemetry_parsing.py

```diff
@@ -1,8 +1,75 @@
 import pytest
+from modules.telemetry.v1.block import BlockHeader
+from modules.telemetry.telemetry_utils import parse_radio_block
 from modules.telemetry.telemetry_utils import is_valid_packet_header
 from modules.telemetry.v1.block import PacketHeader
 from modules.misc.config import load_config
 
+LINE1 = "564133494e490000000c0101010000000200020000000000ac5d00000200030000000000f0c300000200030000000000f0c30000"
+
+
+@pytest.fixture
+def pkt_version() -> int:
+    """
+    returns the packet version as an integer
+    """
+
+    return int("0x" + LINE1[19:21], 16)
+
+
+@pytest.fixture
+def block_header() -> BlockHeader:
+    """
+    returns a blockheader
+    """
+
+    return BlockHeader.from_hex(LINE1[32:40])
+
+
+@pytest.fixture
+def hex_block_contents() -> str:
+    """
+    returns the contents
+    """
+    return LINE1[-16:]
+
+
+@pytest.fixture
+def block_header_error() -> BlockHeader:
```

Comment:
Nitpick but I'd probably call this `bad_block_header` so its not confused as returning an error type.